komodo/symlink/create_links.py

```diff
@@ -30,6 +44,9 @@ def _create_link(src, dst, link_dict):
             return
         os.remove(dst)
         print(f"Existing symlink {dst} moved from {existing_link} to {src}")
+        implicitly_moved_sources = get_implicitly_moved_symlinks(dst, link_dict)
+        for source in implicitly_moved_sources:
+            print(f"{source} was implicitly moved")
```

Comment:
I think I only want the last one, would that the first or last element of the list? And since this is to be sent to a Slack message, I would like to have it formatted as a sentence to read. What happens with the lineshifts from `print` statements is a little hard to predict.

Reply:
I updated it. Can you verify if it looks good now / how you would improve it?